src/main/webapp/pages/testConfiguration/testAdd.jsp

```diff
@@ -589,6 +589,29 @@
             }
         });
     }
+
+    function criticalRangeCheck() {
+        var highCriticalRangeValue, lowCriticalRangeValue;
+        var lowCriticalRange = jQuery("#lowCriticalRange");
+        var highCriticalRange = jQuery("#highCriticalRange");
+        lowCriticalRange.removeClass("error");
+        lowCriticalRangeValue = lowCriticalRange.val();
+        if((lowCriticalRangeValue >= 0) && (lowCriticalRangeValue <= 10) || (lowCriticalRangeValue >= 95)){
```

Comment:
these numbers should not be hardcoded. The critical ranges should be defined in test add and test modify and be results should be checked against them on result entry/validation

Reply:
@CalebSLane jsut for clarification on what you mean by `critical ranges should be defined in test add and test modify`, do you mean in the test add and test modify jsp?